.github/workflows/tests.yml

```diff
@@ -265,10 +265,10 @@ jobs:
         run: yarn workspace ${{ matrix.example }}-example start & npx wait-on -c waitOnConfig.json -t 20000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-username
 
       - name: Run E2E tests against ${{ matrix.example }} example
-        run: yarn workspace e2e test:authenticator
+        run: yarn workspace e2e test:examples
```

Comment:
Are we still run the test:authenticator tests somewhere else?

Reply:
This will run both hooks and authenticator tests:
https://github.com/aws-amplify/amplify-ui/pull/1276/files#diff-67adead8de9872bd465961c6570bce9bc06e7164ea116e45ad405ddf8eff238aR9